Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -41,15 +46,22 @@ + (instancetype)sharedManager;
 
 @implementation PINAnimatedImageManager
 
-+ (void)initialize
++ (void)load
```

Comment:
Traced again with dispatch_async version and that also doesn't show up. Will go with that.